src/provider/element-templates/ElementTemplates.js

```diff
@@ -97,13 +106,51 @@ export default class ElementTemplates {
             version = isUndefined(template.version) ? '_' : template.version;
 
       if (!this._templates[ id ]) {
-        this._templates[ id ] = {};
+        this._templates[ id ] = {
+          latest: template
+        };
       }
 
       this._templates[ id ][ version ] = template;
+
+      const latestVerions = this._templates[ id ].latest.version;
+      if (isUndefined(latestVerions) || template.version > latestVerions) {
+        this._templates[ id ].latest = template;
+      }
```

Comment:
I decided to find and save the latest version during import, as we only import once but potentially check for latest versions often.

This version alias also allows us to use `get(id, 'latest')`, if we wanted to.

Reply:
Does this work with updates (i.e. if additional templates are being loaded (async)?)

Reply:
If the templates are added using `elementTemplates.set`, then yes. This is the only API we currently offer and just allows to remove all previous and add new templates.

It does not work if it was sneakily added into `_templates` externally